packages/contentstack-import/src/import/modules-js/custom-roles.js

```diff
@@ -112,6 +112,15 @@ module.exports = class ImportCustomRoles {
       log(self.config, chalk.green('Custom-roles have been imported successfully!'), 'success');
     } catch (error) {
       fileHelper.writeFileSync(customRolesFailsPath, self.fails);
+              let message_content_type = '';
+        if (error.request !== undefined && JSON.parse(error.request.data).role !== undefined) {
+          if (JSON.parse(error.request.data).role.name) {
+            message_content_type =
+              ' Failed custom-role: ' + JSON.parse(error.request.data).role.name;
+          }
+          error.errorMessage = error.errorMessage + message_content_type;
+        }
+        log(self.config, formatError(error.errorMessage), 'error');
```

Comment:
@cs-raj Could you please check to see if we need this? It has already been addressed above.

Reply:
@aman19K  changes are done and removed the redundant code